index.html

```diff
@@ -25,9 +24,9 @@
               <span class="line"></span>
             </button>
             <ul class="nav-menu flex">
-              <li class="item flex"><a href="#work">Portfolio</a></li>
-              <li class="item flex"><a href="#about">About</a></li>
-              <li class="item flex"><a href="#contact">Contact</a></li>
+              <li class="item flex"><a href="#work" class="nav-link">Portfolio</a></li>
+              <li class="item flex"><a href="#about" class="nav-link">About</a></li>
+              <li class="item flex"><a href="#contact" class="nav-link">Contact</a></li>
```

Comment:
- [ ] Good job on working on the mobile menu requirements, everything works as expected. However, one of the goals of this project is to learn how to follow a design template. kindly make the width of the mobile menu full as it is in the design template 😉.